src/navbar/drawer.tsx

```diff
@@ -106,7 +146,15 @@ const Component = (
 
     return (
         <Wrapper ref={ref} data-testid="drawer">
-            <Container $show={show} $viewHeight={viewHeight}>
+            <Container
+                as="nav"
```

Comment:
we can declare Container as `styled.nav` directly, so there's no need to pass in an additional prop

Reply:
closing as I saw that you've updated the role to `dialog`, which is more appropriate as this is an overlaying element